README.md

```diff
@@ -180,7 +180,7 @@ Please take a look at [Q&A](https://github.com/RobotecAI/rai/discussions/categor
 
 ### Developer Resources
 
-See our [Developer Guide](docs/developer_guide.md).
+See our [Developer Guide](docs/developer_guide.md) for a deeper dive into RAI, including instructions on creating a configuration specifically for your robot.
```

Comment:
I suggest maybe even getting Create your own robot at top level, followed by developers resources

Reply:
This is a good idea, let me introduce these changes in the next week, during docs refactoring 🙇🏼‍♂️